New version: FiniteElementContainers v0.7.0

JuliaRegistrator opened the pull request:
- Registering package: FiniteElementContainers
- Repository: https://github.com/Cthonios/FiniteElementContainers.jl
- Created by: @cmhamel
- Version: v0.7.0
- Commit: 2eb70d6d36349c91c2b63215876be0716d2c65fa
- Git reference: HEAD
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
@JuliaRegistrator register

Release notes:

## Breaking changes

- Changed interface to assemblers to better conform to the SciML ecosystem
- Added Neumann boundary condition assembly methods that are GPU compatable
`````
<!-- END RELEASE NOTES -->

cmhamel commented:
@JuliaRegistrator register

Release notes:

## Breaking changes

- Changed interface to assemblers to better conform to the SciML ecosystem
- Added Neumann boundary condition assembly methods that are GPU compatable

JuliaHubRegistrator committed:
New version: FiniteElementContainers v0.7.0

UUID: d08262e4-672f-4e7f-a976-f2cea5767631
Repo: https://github.com/Cthonios/FiniteElementContainers.jl.git
Tree: ef01ccd0df4bd191e52e18354c8e4bfc4f517cc0

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3

cmhamel commented:
[noblock]
